openerp/addons/openupgrade_records/lib/compare.py

```diff
@@ -202,7 +216,8 @@ def match(match_fields, report_fields, warn=False):
         ]
     for column in old_records:
         # we do not care about removed function fields
-        if column['isfunction'] or column['field'] in IGNORE_FIELDS:
+        if (column['isfunction'] or column['isrelated'] or
```

Comment:
This is a good candidate for being refactored to a method that can be called from the multiple places where you are making this change.